### PR TITLE
BAEL-2809 Get all Data in a table with Hibernate

### PR DESCRIPTION
**Pull request by enpy303**

### CODE REVIEW

File: persistence-modules/hibernate5/src/main/java/com/baeldung/hibernate/findall/FindAll.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+    private Session session;
+
+    public FindAll(Session session) {
+        super();
+        this.session = session;
+    }
+
+    public List<Student> findAllWithJpql() {
+        return session.createQuery("Select a From Student a", Student.class)
```
Comment (echeyne) [inline persistence-modules/hibernate5/src/main/java/com/baeldung/hibernate/findall/FindAll.java]: Can you clean up the capitalization of this query? I believe the JPQL standard is to use the same conventions as SQL/MySQL, isn't it?

File: persistence-modules/hibernate5/src/test/java/com/baeldung/hibernate/findall/FindAllTest.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    private Session session;
+    private Transaction transaction;
+    
+    private FindAll findAll;
+
+    @Before
+    public void setUp() throws IOException {
+
+        session = HibernateUtil.getSessionFactory()
+            .openSession();
```
Comment (echeyne) [inline persistence-modules/hibernate5/src/test/java/com/baeldung/hibernate/findall/FindAllTest.java]: In this case and on line 33 you can safely keep these fluent api calls on the same line. We only break up these calls when they are following out of the predefined page area, or if needed in our articles

Comment (echeyne) [reply persistence-modules/hibernate5/src/test/java/com/baeldung/hibernate/findall/FindAllTest.java]: Same thing for `getResultList()` in `FindAll`

Comment (enpy303) [reply persistence-modules/hibernate5/src/test/java/com/baeldung/hibernate/findall/FindAllTest.java]: Baeldung formatter breaks it down on save. I'll do it here, we need to update that formatter ...

File: persistence-modules/hibernate5/src/main/java/com/baeldung/hibernate/findall/FindAll.java (hunk 1 above)
Comment (enpy303) [reply persistence-modules/hibernate5/src/main/java/com/baeldung/hibernate/findall/FindAll.java]: JPQL doesn't have capitalization convention as far as i know. Some projects use uppercase, others lowercase etc. I guess we should define unique style for our tutorials. I uppercased both keywords ...

**Pull request merged**